.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -15,7 +15,9 @@ body:
     id: reproduction
     attributes:
       label: Code for reproduction
-      description: If possible, please provide a minimum self-contained example.
+      description: |
+        If possible, please provide a minimum self-contained example.
+        This field is automatically formatted as Python code.
       placeholder: Paste your code here
```

```suggestion
      placeholder: Paste your code here (this field is automatically formatted as Python code.)
```
I think it might be better here?

---

sure